packages/plugin-typescript/package.json

```
@@ -23,8 +23,8 @@
   },
   "type": "module",
   "dependencies": {
-    "@code-pushup/models": "0.58.0",
-    "@code-pushup/utils": "0.58.0"
+    "@code-pushup/models": "*",
+    "@code-pushup/utils": "*"
```

Only the current version should be specified here. Nx release takes care of bumping it usually, but I happened to make a release just now while this PR was still open :sweat_smile: If you run into problems, merging in `main` should fix it.

```suggestion
    "@code-pushup/models": "0.59.0",
    "@code-pushup/utils": "0.59.0"
```